### ISSUE TITLE
NO trainset class label 

### ISSUE THREAD

**Issue by 2112529** [bbdjj/CGCOD#1]
Why are there no trainset class labels ?

**Comment by bbdjj** [comment]
Sorry, I have been busy revising my paper recently, and I have left it out. I will sort it out and upload it immediately。

**Comment by bbdjj** [comment]
I have uploaded the training set of camoclass.

**Comment by 2112529** [comment]
Typically, in COD tasks, 1,000 images from the CAMO dataset and 3,040 images from COD10K are used as the training set, while CAMO-TEST, COD10K-TEST, NC4K, and CHAMELEON serve as the test sets. However, the file you provided only contains category annotations for the test set. Why didn't you include the category labels for the training set? Does your model not require training?

Additionally, why does the provided split.py file resemble the style of ChatGPT-generated code so much that it seems to have no practical use?

**Comment by bbdjj** [comment]
First, the camoclass training dataset has been produced and is available at CGCOD Releases. It follows the conventional COD task paradigm and contains 4040 images.

Second, split.py is a small utility script for splitting the camoclass test set. If you encounter any difficulties in splitting the dataset, I can provide a pre-split version of camoclass with image and text sets.

**Comment by bbdjj** [comment]
And the entire code of CGCOD will be released when this paper have been accepted.
